src/html/settings.html

```diff
@@ -4,350 +4,414 @@
-                <div class="description">Remembers the last project you used on each integration or URL. Overrides the default project.</div>
-              </div>
-
-              <div>
-                <select id="remember-project-per">
-                  <option value="false">Use global default</option>
-                  <option value="service">Service</option>
-                  <option value="url">URL</option>
-                </select>
-              </div>
+      <div class="section">
+        <h3>Settings</h3>
+        <ul class="nav-links">
+          <li class="nav-link" data-tab="general">
+            <div class="nav-icon">
+              <svg width="16" height="16" viewBox="0 0 24 24">
+                <path fill="#fff" d="M24 14v-4l-3-1-1-2 2-2-3-3-2 2-2-1-1-3h-4L9 3 7 4 5 2 2 5l2 2-1 2-3 1v4l3 1 1 2-2 2 3 3 2-2 2 1 1 3h4l1-3 2-1 2 2 3-3-2-2 1-2 3-1zm-12 2a4 4 0 110-8 4 4 0 010 8z"/>
+              </svg>
             </div>
-
-            <div class="field">
-              <div class="field__label">
-                <label for="enable-auto-tagging">Enable automatic tagging</label>
-                <div class="description">If an integration supports tags or labels, automatically add them to the timer and
-                  create them in Toggl. Not available in all integrations.</div>
-              </div>
-
-              <div>
-                <input type="checkbox" id="enable-auto-tagging">
-              </div>
+            <span>General</span>
+          </li>
+          <li class="nav-link" data-tab="pomodoro">
+            <div class="nav-icon">
+              <svg width="16" height="16" viewBox="0 0 26 26">
+                <defs>
+                  <clipPath id="a">
+                    <path d="M0 0h26v26H0z"/>
+                  </clipPath>
+                </defs>
+                <g clip-path="url(#a)">
+                  <path fill="#fff" fill-rule="evenodd" d="M4.7 4.7C1.4 6.4 0 8.9 0 14c0 6.8 3.2 11.8 13 12 9.5 0 13-3 13-13a9 9 0 00-4.7-8.4c-.8.7-1.4 1-1.3 1.4l3 5c-2.1 1.4-9.8-2-10-2-3 2.3-8.7 1.6-9 1-.4-1 2.3-2.5 3-4-.9-.2-1.7-.8-2.3-1.3zm2.2-1.9c1.6-.2 3.3.1 5.1.2.2 0 .8-3 1-3 .3 0 1.8.4 2 1 .1.4-.2 2 0 2 .9 0 3.3-1 5 0l.2.2c-.9.3-2.2 1-3.2 1.8-.8.8 3.6 4.4 2 4-4.1-1-5.5-2.4-6-2a11 11 0 01-6 2c1.5-1.5 3.3-4 3-4a7.5 7.5 0 01-3.1-2.2z"/>
+                </g>
+              </svg>
             </div>
-
-            <hr />
-
-            <div class="field">
-              <div class="field__label">
-                <label for="start_automatically">Start timer automatically</label>
-                <div class="description">Starts the timer with the most recent task when the browser is opened</div>
-              </div>
-              <div>
-                <input type="checkbox" id="start_automatically">
-              </div>
+            <span>Pomodoro</span>
+          </li>
+          <li class="nav-link" data-tab="integrations">
```

Comment:
nitpicking but integrations should be higher than pomodoro 😅 it's the most important thing, really. I wonder if we should have a hint banner inside the `General` settings page that if you want to enable Toggl Button in tools, go there.

Reply:
sure, will think about how to add it to `General` :thinking:

Reply:
#1614